balancer-labs/frontend-v2
# Hotfix/1.61.7

# Description

The limit wasn't being set correctly for single swaps (reported by Copper). I've added a fix for this.

## Type of change


## How should this be tested?

On Goerli and current release try swapping WETH>WBTC, ExactOut 0.0001. You will see a 507 error which is SWAP_LIMIT. If you track this back you will see the limit is being set using the amount out but for ExactOut it should use amount in.
The same trade on hotfix should work.

## Visual context

Please provide any relevant visual context for UI changes or additions. This could be static screenshots or a loom screencast.

## Checklist:
